specs/parser/lexical/lists_spec.js

```diff
@@ -1,36 +1,32 @@
-import Immutable from 'immutable';
 import { parserFor } from '../../../app/oz/parser';
 import lexicalGrammar from '../../../app/oz/grammar/lexical.nearley';
+import lexical from '../../samples/lexical';
 
 const parse = parserFor(lexicalGrammar);
 
 describe('Parsing lists', () => {
-  beforeEach(() => {
```

Comment:
I think that the custom equality matcher is actually needed if you want true deep equality checking here.